AndroidSDKCore/src/main/java/com/leanplum/internal/RequestSequenceRecorder.java

```diff
@@ -0,0 +1,28 @@
+package com.leanplum.internal;
+
+/**
```

Comment:
java format: use the Google Java Formatter to auto format this file, CMD + OPT + L

Reply:
done